MementoMori/MementoMori.Tests/UnitTests/ServiceTests/DeckHelperTests.cs

```diff
@@ -101,5 +187,49 @@ public void UpdateDeck_RemovesCards()
 
             Assert.Empty(context.Cards);
         }
+
+        [Fact]
+        public async Task CreateDeck_CreatesNewDeckSuccessfully()
+        {
+            var context = CreateDbContext();
+            var helper = new DeckHelper(context);
+
+            var deck = createTestDeck1();
+
+            Guid requesterId = Guid.NewGuid();
+
+            Guid savedDeckId = await helper.CreateDeckAsync(deck, requesterId);
+
+            var savedDeck = context.Decks.Include(d => d.Cards).FirstOrDefault(d => d.Id == savedDeckId);
+
+            Assert.NotNull(savedDeck);
+            Assert.Equal(requesterId, savedDeck.CreatorId);
+            Assert.Equal(deck.Deck.Title, savedDeck.Title);
+            Assert.Null(savedDeck.Description);
+            Assert.True(savedDeck.isPublic);
+            Assert.Equal(deck.Deck.Tags, savedDeck.Tags);
+            Assert.NotNull(savedDeck.Cards);
+            Assert.Equal(deck.NewCards?.Length, savedDeck.Cards.Count);
+        }
+        [Fact]
+        public async Task DeleteDeck_DeleteDeckSuccessfully()
+        {
+            var context = CreateDbContext();
+            var helper = new DeckHelper(context);
+
+            var deck = createTestDeck2();
```

Comment:
Why not reuse createDeck1 here?

Reply:
For simplicity of reading as you wont need to check other methods to look up how the test works, avoids potential errors from it.